lib/contracts/flowdock-thread.ts

```diff
@@ -20,13 +20,19 @@ export const flowdockThread: ContractDefinition = {
 						id: {
 							type: 'string',
 						},
+						urls: {
+							type: 'array',
+							items: {
+								type: 'string',
+							},
+						},
 						flow: {
```

Comment:
@iamsolankiamit Is `flow` needed? I'm guessing this is now represented with the separate `flow` contract?

Reply:
right, i missed this! thanks